sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ForLoopIncrementSignCheck.cs

```diff
@@ -6,13 +6,23 @@ public class ForLoopIncrementSignCheck
     {
         public void TestMethod(int x, int y, int z)
```

Comment:
I would also add 
- a `for` with 2 iterators
- a `for` with 2 initializers

Reply:
Done.